py-rse/correct.Rmd

```diff
@@ -174,17 +179,17 @@ A single unit test will typically have:
 -   a [fixture][fixture],
     which is the thing being tested (e.g., an array of numbers);
 -   an [actual result][actual-result],
-    which is what the code produces when given the fixture; and
+    which is what the code produces when given the fixture;
+    and
 -   an [expected result][expected-result]
     that the actual result is compared to.
 
-In defining/creating an appropriate test fixture,
-a common approach is to use a subset or smaller version of the data
-that the function will typically process.
-For instance, in order to write a unit test for the `count_words` function,
-an appropriate fixture would be a small body of text
-for which we know the frequency of each word.
-The poem Risk by Anais Nin is a good choice,
+The fixture is typically a subset or smaller version of the data
+the function will typically process.
+For instance,
+in order to write a unit test for the `count_words` function,
+we could use a piece of text small enough for us to count word frequencies by hand.
+The poem "Risk" by Anaïs Nin is a good choice:
```

Comment:
Do we need to make it clearer here that `data/risk.txt` is a file we just created? It wasn't something that was in the `data/` directory all along.

Reply:
Clarified.